### ISSUE TITLE
General overview question on plugin stability & lifespan

### ISSUE THREAD

**Issue by oscarjlep** [FranckFreiburger/http-vue-loader#14]
I really love what you have come up with in here! I'v been looking for simple solution for loading & parsing vue-files directrly from browser with runtime-compilation and apparently you came up with solution! 

1)I'm only wondering how stable do you consider this solution to be? 
2) I'v tried it with IE11, Chrome & Firefox with no problems, but do you think the core solution of library has lifespan for upcoming browser updates & vue-updates (== "is it generic solution")? 
3) Well, not actually a question : Apparently it's not using runtime-compilation, but simply registers script-section to Vue - simple & clever!
4) ... most interesting question for myself: Do you plan to continue updating this plugin, as it seems to be one of the most interesting ones I have seen?
5) As I have problem introducing webpack-stack to rather hard-to-maintain-and-configure environment, I wonder what are the real cons on using http-vue-loader as runtime-solution? Would you strongly oppose using http-vue-loader in actual production use and why (or would you give at least "careful green" :) ?

**Comment by FranckFreiburger** [comment]
Hi,

First of all, use webpack, it is a fantastic tool, learning curve (for complex setup) is quite hard, but the result is worth the effort !
The aim of http-vue-loader is for quickly testing, prototyping and mockup without having to install and setup webpack.

cons:
- not heavily tested.
- since everything is done at runtime, http-vue-loader may be quite slow.
- incomplete CSS processing (scoped CSS).
- unsupported nested selectors in scoped styles (>>> operator).

pros:
- quite stable
- http-vue-loader does not do much, it should be easy to maintain.

then, use in production at your own risk.

**Comment by oscarjlep** [comment]
Thanks for an answer!

My main issue with webpack is that I have quite complicated building process which I'm unable to extend with webpack. Issue is not only how development build is done, but also how release builds are done. Having webpack would require lot's of work on multiple servers as well as introducing it to building pipeline on both production sites and development tools, which seems like too much effort at current state. I really hate my workspace setup, but it's not something I have control over (Java & Eclipse & Tomcat & Liferay 6.2). It's such a bloat... I would be glad to start using Webpack if I were to use Laravel or .NET MVC .

Speed of run-time processing isn't an issue for me, but scoped CSS is something I'd love to use. I'v been experimenting with http-vue-loader for couple of days now and I stumbled to first cross-browser issue today. Apparently scoped use doesn't work on IE11 / Edge. I'm likely to experiment if this is only due to using attribute selector instead of generating scope by ID or class, as I had no problem when I used custom CSSX -injector solution for scoping.
